packages/python-plum/PKGBUILD

```diff
@@ -0,0 +1,30 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+# This file is based on the AUR package by Caleb Maclennan <caleb@alerque.com>
+
+_pyname=plum
+pkgname=python-${_pyname,,}
+pkgver=0.8.5
+pkgrel=1
+pkgdesc='Pack/Unpack Memory'
+arch=(any)
+url="https://$_pyname-py.readthedocs.io/en/latest/"
+license=(MIT)
+depends=(python)
+makedepends=(python-{build,installer}
+             python-setuptools
+             python-wheel)
+_archive="$_pyname-$pkgver"
+source=("https://gitlab.com/dangass/$_pyname/-/archive/$pkgver/$_archive.tar.bz2")
```

Comment:
use https://pypi.org/project/plum-py/ and https://github.com/BlackArch/blackarch-pkgbuilds/blob/master/PKGBUILD-python-lib-setuptools

Reply:
Changed. I changed also the pkg name as `python-plum-py`. Is it possible to change the directory of the PKGBUILD as `python-plum-py` by GitHub WebUI?